git-dit.1.md

```diff
@@ -95,6 +96,223 @@ the former one is especially useful for patch-sets.
+The usual rules for commit messages also apply to issues: the first line will be
+the subject line and should contain a summary or appropriate title; the second
+line should be blank, followed by paragraphs of texts or blocks of trailers.
+The trailers may be used for transporting metadata.
+Once the message is written and the editor is closed, a new issue will be
+created and git-dit will print the issue's id.
+
+For others to see the issue, the issue has to be pushed to a public repository.
+The command
+
+    git dit push origin <id>
+
+will push the issue with the id provided by the user to the remote "origin".
+Alternatively, all local updates, including new issues, may be pushed to the
+remote "origin" using the command
+
+    git dit push origin
+
+## Viewing issues
+
+Issues can both be listed and viewed.
+The command
+
+    git dit list
+
+lets the user view the issues known to the repository.
+The list contains each issue's id, which the user may copy, e.g. into her
+clipboard, for further use.
+
+For example, an issue may be shown using a previously obtained id using the
+command
+
+    git dit show <issue-id>
+
+This command displays the messages of an issue.
+Multiple output formats are supported.
+Most of them will contain the messages' ids.
+Again, users may copy a message's id for further use, e.g. for replying to that
+message.
+
+## Adding information and metadata to an issue
+
+Users may add information in the form of text and trailers to an issue by
+replying to an issue message.
+The command
+
+    git dit reply <message-id>
+
+spawns an editor for composing a reply to the message provided.
+The message is handled (mostly) like a regular commit message:
+the first line is a subject line;
+the second line should be empty;
+lines starting with a '#' are removed.
+Trailers should go to the end of a message.
+
+After the message is saved and the editor closed, the message will be added to
+the issue.
+It should now be visible via the "show" subcommand.
+
+Currently, the new message only lives in the local repository.
+In order to make it visible to others, users will usually want to push the new
+message using the "push" subcommand.
+
+Note that new metadata, e.g. the sate of an issue, added via a message is not
+immediately adopted.
+Rather it should be considered a proposal for a metadata change.
+The remote repository's maintainer and possibly other moderating parties may
+apply those proposed changes by updating the issue's "head" reference.
+
+## Managing the state and other metadata of an issue
+
+The "head" reference of an issue represents the "upstream state" of an issue.
+The meatadata, e.g. status, of an issue is computed by accumulating the metadata
+of messages from the head reference to the issue's initial message, only
+following the first parent of each message.
+
+Consider a bug-report consisting of the following tree of messages:
+
+    A
+    |
+    B <- head
+    |\
+    C D
+    |
+    E
+    |\
+    F G
```

Comment:
Please clearify (in text or otherwise) which node in this graph represents the oldest commit!